assets/src/edit-story/app/media/utils/useMediaUploadQueue/index.js

```diff
@@ -157,7 +157,7 @@ function useMediaUploadQueue() {
     async ({ newResource, posterFileName, newPosterFile, resource, id }) => {
       try {
         const { poster, posterId } = await uploadVideoPoster(
-          newResource.id,
+          newResource?.id,
```

Comment:
What's the rationale for this?

If `newResource` does not exist, then we have a bigger problem. It means `uploadVideoPoster()` could be called with `null`, which would not work and probably cause some HTTP errors.

So if `newResource.id` does not exist we should not be calling `uploadVideoPoster`